dynomitemanager/src/main/java/com/netflix/dynomitemanager/identity/DefaultVpcInstanceEnvIdentity.java

```diff
@@ -0,0 +1,22 @@
+package com.netflix.dynomitemanager.identity;
```

Comment:
With the latest PR I did, Dynomite-manager automatically checks the environment it is running.
See [AwsInstanceEnvIdentity](https://github.com/Netflix/dynomite-manager/blob/dev/dynomitemanager/src/main/java/com/netflix/dynomitemanager/identity/AwsInstanceEnvIdentity.java)

Using the DefaultVpcInstanceEnvIdentity means that (a) we statically bind the environment and (b) we cannot run clusters in hybrid mode (some nodes in VPC and some in EC2 classic). I think if you bind the AwsInstanceEnvIdentity it should work in VPC for you.